profile/rwahs.xml

```diff
@@ -3181,11 +3205,8 @@
             <placement code="idno">
               <bundle>idno</bundle>
             </placement>
-            <placement code="ca_attribute_georeference">
-              <bundle>ca_attribute_georeference</bundle>
-            </placement>
-            <placement code="ca_attribute_description">
-              <bundle>ca_attribute_description</bundle>
+            <placement code="ca_attribute_Description">
+              <bundle>ca_attribute_Description</bundle>
```

Comment:
Found by the tests - I renamed description to Description following our convention, but didn't change all the bundle placements for it